boilerplate/package.json

```diff
@@ -3,17 +3,15 @@
   "version": "1.0.0",
   "description": "A boilerplate that allows running a buildless app.",
   "homepage": "https://zapier.com/",
-  "author": "Zapier <partners@zapier.com>",
+  "author": "Zapier Engineering <engineering@zapier.com>",
```

Comment:
We usually always ask for people to reach out at partners@zapier.com. Any reason for this change (and should we change in the other packages)?

Reply:
That's a good point. I reverted it back. I made this change only because the author of our CLI packages on npm is engineering@zapier.com.